### PR TITLE
Rename InvokeAsync to SendAsync on the server

### PR DESCRIPTION
**Pull request by BrennanConroy**
#1300

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Core/IClientProxy.cs [hunk 1]
```diff
@@ -8,11 +8,11 @@ namespace Microsoft.AspNetCore.SignalR
     public interface IClientProxy
     {
         /// <summary>
-        /// Invokes a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
+        /// Fire-and-forget a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
```
Comment (mikaelm12) [inline src/Microsoft.AspNetCore.SignalR.Core/IClientProxy.cs]: I do agree that this should be changed but this doesn't read well to me.

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Core/IClientProxy.cs]: I'd change it back. It does invoke a method. Just mention that it doesn't wait for the other side to send a reply.

**Comment by davidfowl** [comment]
1. Why is Travis failing?
1. Are we going to change the HubLifetimeManager methods?

**Comment by BrennanConroy** [comment]
1. One of our flaky tests
2. We probably should for consistency

**Pull request merged**